tests/units/feature_extraction/test_feature_calculations.py

```diff
@@ -284,6 +300,11 @@ def test_augmented_dickey_fuller(self):
         for index, val in res_value_error:
             self.assertIsNaN(val)
 
+        # Should return NaN if "attr" is unknown
+        res_attr_error = augmented_dickey_fuller(x=x, param=[{"autolag": "AIC", "attr": ""}])
+        for index, val in res_attr_error:
```

Comment:
Wouldn't it be better to throw some warning?'if we do not know the setting?

Reply:
This just preserves the old behaviour. If it is preferable to raise
errors (instead of return NaN) in such case, I'll fix it so.

Reply:
Right, correct!